download-basket/src/main/java/downloadbasket/data/ZipDownloadDetails.java

```diff
@@ -0,0 +1,22 @@
+package downloadbasket.data;
+
+public class ZipDownloadDetails {
+	private String fileName = "";
+	private String layerName = "";
+
+	public String getFileName() {
+		return fileName;
+	}
+
+	public void setFileName(String fileName) {
+		fileName = fileName;
+	}
+
+	public String getLayerName() {
+		return layerName;
+	}
+
+	public void setLayerName(String layerName) {
+		layerName = layerName;
+	}
+}
```

Comment:
I made another JUnit-test for you. Check if the asserts work in your environment:

```
package downloadbasket.data;

import org.junit.Test;

import static org.junit.Assert.*;

public class ZipDownloadDetailsTest {

    final String TESTING = "hello world";
    @Test
    public void testSetFileName() throws Exception {
        ZipDownloadDetails details = new ZipDownloadDetails();
        details.setFileName(TESTING);
        assertEquals(details.getFileName(), TESTING);

    }

    @Test
    public void testSetLayerName() throws Exception {
        ZipDownloadDetails details = new ZipDownloadDetails();
        details.setLayerName(TESTING);
        assertEquals(details.getLayerName(), TESTING);

    }
}
```